lib/types/cmaClient.types.ts

```diff
@@ -2,6 +2,7 @@ import { PlainClientAPI } from 'contentful-management'
 
 export type CMAClient = {
   appAction: Pick<PlainClientAPI['appAction'], 'get' | 'getMany' | 'getManyForEnvironment'>
+  appActionCall: Pick<PlainClientAPI['appActionCall'], 'create'>
```

Comment:
I specifically didn't include that because we will be doing `await cma.appActionCall.create('id')` which should resolve with whatever is returned by the `callDetails`. For DX, they call an action and get a response. We should be handling the messy part :)